inspec.yml

```diff
@@ -8,55 +8,602 @@ summary: "CMS ARS 5.0 Overlay InSpec Validation Profile for RHEL8 STIG"
 version: 1.3.0
 inspec_version: ">= 4.0"
 
-
-
 depends:
 - name: redhat-enterprise-linux-8-stig-baseline
-  url: https://github.com/CMSgov/redhat-enterprise-linux-8-stig-baseline/archive/v2.1.0.tar.gz
-
+  path: /Users/ssayed/redhat8-baseline/redhat-enterprise-linux-8-stig-baseline
```

Comment:
Since you're making this the official code for the overlay, you should stop using local system paths for things -- uncomment out that `url` tag instead